tools/locate_bins.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh -e
+# Copyright (c) 2017-2022, Mudita Sp. z.o.o. All rights reserved.
+# For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+have_command () {
+  eval "$1 --version >> /dev/null 2>&1"
+  if [ $? -ne 0 ]
+  then
+    echo "false"
+  fi
+  echo "true"
+}
+
+eval "sudo -n echo >> /dev/null 2>&1"
+if [ $? -eq 0 ]; then
+  export have_sudo="true"
+else
+  export have_sudo="false"
+fi
+
+get_command () {
+  path_val=""
+  if $(have_command $1) ]; then
+    path_val=$(command -v $1)
+  elif $have_sudo; then
+    path_val=$(sudo -l $1 2>&1)
+    if [ "$path_val" ]; then
+      if [ -e "$path_val" ]; then
+        cmd_val="sudo $path_val"
+      else
+        return 1
+      fi
+    else
+      return 2
+    fi
+  else
+    return 1
+  fi
+  eval "export -p call_${1}=\"$path_val\""
+  return 0
+}
+
+require_commands () {
+  for name in $@; do
+    get_command $name
+    case $? in
+      1)
+        echo "Error: $name is not installed, please use 'sudo apt install' for install required tool"
```

Comment:
We do not limit ourselves to apt-based Linux. Just write "follow your OS installation guide" or something like that.

Reply:
I'm OK with that.  The text was lifted from the other scripts.